index.html

```diff
@@ -2396,9 +2397,23 @@ <h2 id="AudioParam">
               </dt>
               <dd>
                 The value the parameter will exponentially ramp to at the given
-                time. A NotSupportedError exception MUST be thrown if this
-                value is less than or equal to 0, or if the value at the time
-                of the previous event is less than or equal to 0.
+                time. A NotSupportedError exception MUST be thrown if any
```

Comment:
Although I did this in this patch, couldn't we do this the same way that "MUST" is magically highlighted (by respec)?